src/docs/issue-platform.mdx

```diff
@@ -29,6 +29,8 @@ You may also set these optional attributes:
 - `noise_config`: A `NoiseConfig` that allows you to set policy on how many times a fingerprint must occur in a period of time before an issue is created. NoiseConfig has two attributes:
   - `ignore_limit`: How many occurrences need to occur before we create a new issue
   - `expiry_time`: The period of time the occurrences must occur in before we reset the count
+- `enable_auto_resolve`: Whether detector controlled resolution should be enabled for this issue type. By default this attribute is set to true.
```

Comment:
are we calling it detector-controlled resolution? that name doesn't mean much without context